setup.cfg

```diff
@@ -6,14 +6,41 @@ python_requires = >=3.9
 packages = find_namespace:
 include_package_data = true
 install_requires =
+    great-expectations>=1.3.1
     apache-airflow>=2.1
-    great-expectations[snowflake,postgresql,mssql,bigquery,athena,spark,gcp,azure,s3]>=1.3.1
     setuptools>=41.0.0
 
 [options.extras_require]
+athena =
+    great-expectations[athena]>=1.3.1
+azure =
+    great-expectations[azure]>=1.3.1
+bigquery =
+    great-expectations[bigquery]>=1.3.1
+lint =
+    mypy==1.14.1
+    ruff==0.8.3
+    pytest==8.3.4
+    pytest-mock==3.14.0
+    great-expectations[spark, spark-connect]>=1.3.1
```

Comment:
Do we really need `spark` in `lint`?

Reply:
Good question. We do because spark dataframes are in the type signature of at least one of the operators. I don't see a way around it, but LMK if you do.

Reply:
We do support pandas. And that is part of the base gx install, so it's already included. I'll update the description on this PR, since these are good things to be clear about.